### PR TITLE
ECAL AlcaReco datasets definition 

### PR DESCRIPTION
**Pull request by emanueledimarco**
This is the equivalent of #7870, done in 75X and fixing the includes of the reconstruction sequences for comics.
@ferriff @crovelli @emanueledimarco @argiro @cerminar  would like to follow the PR

### CODE REVIEW

**Comment by emanueledimarco** [comment]
and of course @shervin86 should follow the PR, since he is the main developer of it

File: Calibration/EcalAlCaRecoProducers/src/ValueMapTraslator.cc [hunk 1]
```diff
@@ -0,0 +1,216 @@
...
+{
+   using namespace edm;
+   std::vector<value_t>  valueVector;
+   std::auto_ptr<Map_t> valueVectorPtr(new Map_t());
+
+   //------------------------------ 
+   iEvent.getByLabel(referenceCollectionTAG, referenceHandle);
+   iEvent.getByLabel(oldreferenceCollectionTAG, oldreferenceHandle);
+   iEvent.getByLabel(inputCollectionTAG, inputHandle);
+
```
Comment (davidlange6) [inline Calibration/EcalAlCaRecoProducers/src/ValueMapTraslator.cc]: needs consumes to be done.

Comment (emanueledimarco) [inline Calibration/EcalAlCaRecoProducers/src/ValueMapTraslator.cc]: @davidlange6 done

**Comment by shervin86** [comment]
I have the fixes. This PR can be closed in favour of PR  #8306

**Comment by emanueledimarco** [comment]
Closed.

**Pull request closed without merging**